extensions/amazon-lambda-resteasy/maven/src/test/resources/projects/configure-existing-template/template.yaml

```diff
@@ -0,0 +1,29 @@
+Transform: 'AWS::Serverless-2016-10-31'
+Resources:
+
+  ThumbnailFunction:
+    # This resource creates a Lambda function.
+    Type: 'AWS::Serverless::Function'
+
+    Properties:
+
+      # This function uses the Nodejs v6.10 runtime.
+      Runtime: nodejs6.10
```

Comment:
Typo?  nodejs6.10?

Reply:
yes and no.  it gets updated by the mojo but i'll fix the template.